ChainReactionBindings/Bindings/WishlistPageBindings.cs

```diff
@@ -86,5 +86,38 @@ public void WhenIClickConfirmButton()
 		{
 			page.ConfirmDelete();
 		}
+
+		[Then(@"I see empty wishlist with message ""(.*)""")]
+		public void ThenISeeEmptyWishlist(string message)
+		{
+			Assert.AreEqual(message, page.EmptyWishlistElement.Text);
+		}
+
+		[Then(@"I see ""(.*)"" items in wishlist")]
+		public void SeeItemsInWishlist(int count)
+		{
+			Assert.AreEqual(count, page.WishlistContainerElement.FindElements(By.ClassName("wishlist_product_container")).Count);
+		}
+
+		[When(@"I click Remove button on item with name ""(.*)""")]
+		public void ClickRemoveButtonOnItemWithName(string name)
+		{
+			var el = page.FindItem(name);

```

Comment:
Please rename FindItem to GetItem, it makes  more sense

Reply:
done